src/BPEssentials/Models/KitHandler.cs

```diff
@@ -0,0 +1,28 @@
+﻿using BPEssentials.FileHandler;
+using System.Collections.Generic;
+
+namespace BPEssentials
+{
+    public class KitHandler : ExpandableFileHandler<KitHandler.JsonModel>
+    {
+        public KitHandler()
+        {
+            Name = "kit";
+            FileExtension = "json";
+            FilesDirectory = Core.Instance.Paths.KitsFolder;
+        }
+        public class JsonModel : IExpandableFile, IExpandableFileDelayable, IExpandableFileHasPrice
+        {
+            public string Name { get; set; }
+            public bool Disabled { get; set; }
+            public int Price { get; set; }
+            public int Delay { get; set; }
+            public List<Kits_Item> Items { get; set; } = new List<Kits_Item>();
+        }
+        public class Kits_Item
```

Comment:
Don't use snake_case in types, unless in defines. (eg `#DEFINE HELLO_WORLD`)
```suggestion
        public class KitsItem
```